src/main/java/org/gitlab4j/api/models/MembershipSourceType.java

```diff
@@ -1,31 +1,38 @@
 package org.gitlab4j.api.models;
 
-import org.gitlab4j.api.utils.JacksonJsonEnumHelper;
-
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonValue;
 
-public enum MembershipSourceType {
+import java.util.Locale;
 
-    PROJECT,
+public enum MembershipSourceType {
+    PROJECT("Project"),
 
     /** Representing a group */
-    NAMESPACE;
+    NAMESPACE("Namespace");
 
-    private static JacksonJsonEnumHelper<MembershipSourceType> enumHelper = new JacksonJsonEnumHelper<>(MembershipSourceType.class);
+    public final String name;
+
+    MembershipSourceType(String name) {
```

Comment:
We don't need to add an attribute name here. We should use a different JacksonJsonEnumHelper constructor. See `AwardableType` for example. 
Can you take care of this.

Reply:
Done: https://github.com/gitlab4j/gitlab4j-api/pull/816